lib/widgets/upload/ad_form_title.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:flutter/material.dart';
+
+class AdFormTitle extends StatelessWidget {
+  final String title;
+
+  AdFormTitle(this.title);
+
+  @override
+  Widget build(BuildContext context) {
+    return Text(title);
+  }
+}
```

Comment:
Is this really necessary? All it does is return Text widget might as well use the Text widget directly.
https://www.youtube.com/watch?v=-AQfQFcXac8&ab_channel=javidx9

Reply:
My idea is that we can extend this widget later with design, padding etc.